README.md

```diff
@@ -1,2 +1,5 @@
 # Juntos
 Juntos is a web application that lets you co watch streaming service such as youtube. More services to be added in future such as Netflix, Prime, HBO NOW ...etc.
+
+* Artefacts <br>
+[PRD](/prd/Juntos_PRD.pdf) <br>
```

Comment:
Can we loose the `<br>`s

Sorry for the confusion . In https://github.com/pesto-students/juntos-ninja-2-beta/pull/8#discussion_r641372427 I meant that we make the PRD line a bullet point. 'Artefacts' works as a heading as it would have multiple links below it.